spec/dummy/db/schema.rb

```diff
@@ -11,21 +11,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190909093455) do
+ActiveRecord::Schema.define(version: 20190805152649) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "ad_renewal_letters_exports", force: :cascade do |t|
-    t.date     "expires_on"
-    t.string   "file_name"
-    t.integer  "number_of_letters"
-    t.string   "printed_by"
-    t.date     "printed_on"
-    t.datetime "created_at",        null: false
-    t.datetime "updated_at",        null: false
-  end
```

Comment:
I think we shouldn't kill this 🤔

Reply:
I'm not, Rails is!

```bash
cd waste-exemptions-engine
git pull
bundle install
RAILS_ENV=test bundle exec rake db:drop
RAILS_ENV=test bundle exec rake db:create

Reply:
We should probably move that migration here?

Reply:
No, I think it needs to stay where it is and this _does_ need to be deleted. Unless we have functionality in the engine that depends on this table existing (which based on our tests would suggest we don't as they pass with this missing from the DB) we shouldn't have it in the engine.

We've currently gone with a convention of keeping back office only related migrations in the back office (see https://github.com/DEFRA/waste-exemptions-back-office/tree/master/db/migrate). This sounds like it fits in that category.